### DISCUSSION TITLE
Error with Chinese caracter

### DISCUSSION THREAD

**Discussion by ShamanMap** [YaoFANGUK/video-subtitle-extractor#347] [Q&A] [answered]
When I download and run the GPU version.

I get this strange error. Could you please help?

![Warning - GitHub_ QPT-Family_QPT 1727623693](https://github.com/user-attachments/assets/10fb6ff7-2445-411c-baed-c6df466d2b9e)

**Comment by zndx1002070101** [comment]
我也有同样的问题！

**Comment by eritpchy** [comment, marked as answer]
Remove space char in "SOFT SOURCE" directory
